newIDE/app/src/UI/SortableVirtualizedItemList/index.js

```diff
@@ -102,71 +135,60 @@ export default class SortableVirtualizedItemList<Item> extends React.Component<
 
```

Comment:
Nitpicking: as we have now renderItemRow, let's move these nameBeingEdited and itemName to renderItemRow.

Reply:
This will reduce the number of arguments that you need to send to renderItemRow, which is great because as it's a renderXXX function, it must be autonomous as much as possible (could even be turned into its own component later - but I don't think it's worth doing it now).